styles/colors.css

```diff
@@ -1,3 +1,4 @@
 :root {
   --color-container: rgba(0, 0, 0, 5%);
+  --color-button: rgb(98, 185, 178);
```

Comment:
I am not sure, but i think it might be more handy to write the color name instead of container and button?

Reply:
Thank you. I thought about that but the one color is just like a shade and its easier for me to remember, when I name them in that way. But probably you are right and I should give them color names. Could be too confusing in a bigger project. Thanks :)